src/roiextractors/extractors/memmapextractors/memmapextractors.py

```diff
@@ -71,36 +69,20 @@ def write_imaging(imaging_extractor: ImagingExtractor, save_path: PathType = Non
         Parameters
         ----------
         imaging: ImagingExtractor object
-            The EXTRACT segmentation object from which an EXTRACT native format
-            file has to be generated.
         save_path: str
             path to save the native format.
         overwrite: bool
             If True and save_path is existing, it is overwritten
         """
         imaging = imaging_extractor
-        video_to_save = np.memmap(
+        video_data_to_save = imaging.get_frames()[:]
```

Comment:
May want to use `psutil` to ensure there is actually enough memory available to perform this operation, otherwise the users computer will crash.

Reply:
Also, `tqdm` is always nice, I'd restore that functionality here.

Reply:
Good idea.
I am pretty sure I saw that use of `psutil` somewhere over here but can't find it, am I imaging things? 

I guess you are envisioning something like that is available on the write method of `TiffImagingExtractor`?

Reply:
> I am pretty sure I saw that use of psutil somewhere over here but can't find it, am I imaging things?

https://github.com/catalystneuro/nwb-conversion-tools/blob/a0e5c3b564ff013668f01ceb91bdfc285b2efbda/src/nwb_conversion_tools/datainterfaces/behavior/movie/moviedatainterface.py#L227-L233


> I guess you are envisioning something like that is available on the write method of TiffImagingExtractor?

The red parts of this changelog show how tqdm was used here already. More sophisticated approaches might use a `hdmf.data_utils.GenericDataChunkIterator` which comes along with `tqdm` already, but that might be a bit much to do here.

Reply:
What is that 70 doing there in:
`uncompressed_estimate = Path(file).stat().st_size * 70`

Is it assuming a compression ratio of .7 for the file or something like that?

Reply:
Ohh uhh you can probably ignore that for these cases. That was a manual scalar inflation value I got from experimenting on with loading big movie files from OpenCV. But for all the imaging extractors here, the data is already more or less uncompressed

Reply:
All right, check out the new changes.